kong/db/strategies/postgres/connector.lua

```diff
@@ -148,7 +148,24 @@ local function connect(config)
   end
 
   if connection.sock:getreusedtimes() == 0 then
-    ok, err = connection:query("SET TIME ZONE 'UTC';");
+    if not config.schema or config.schema == "" then
```

Comment:
We assume that `config.schema` is not `nil` everywhere else in this patch. Plus given it is retrieved from the configuration object, it should not in the first place. In the worst case, keeping the `schema = kong_config.pg_schema or ""` in `new()` would be a good idea (e.g. when instantiating by hand in tests or such) and guarantee that we can get rid of this part of the condition.

Reply:
Changed that.